tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -723,7 +736,11 @@ protected void updateViewport() {
   protected void onScrollChanged( int l, int t, int oldl, int oldt ) {
     super.onScrollChanged( l, t, oldl, oldt );
     updateViewport();
-    requestThrottledRender();
+    if( mShouldRenderWhilePanning ) {
+      requestRender();
```

Comment:
Just noticed - should this be `requestSafeRender`, in case it's flinging or programmatically panning (e.g., `slideTo`)?